pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
```

Comment:
I think I'd at least like to see overlapping interfaces defined and implemented.  That prevents ripples, but makes for compatible code.